data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,57 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	(Grab a seed.)`
+					goto seed
+				`	(Grab a seashell.)`
+					goto pearl
+				`	(Grab a feather.)`
+			`	Once you have the feather in your hands, the organizers tell you to go through the path on the right, and so you do.`
+			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it, while keeping your eyes on the small stage in front of you. After the audience is all here, the curtains open, to reveal many small figures made out of what looks like paper, in a style that loosely resembles origami.`
+			`	Entering a clearing, a Saryd huntress with gray hair stalks a large bird resting on a rock. She moves on the borders of the clearing, hiding behind bushes and small trees, looking for the best spot from which to snipe at the bird, her bow and arrow ready in her hands. It's only when she is nearing a large entrance to the clearing, on the opposite side of where she entered, that she finds the spot, and takes aim.`
+			`	She is disrupted when she hears a loud roar, and jumps to the ground to dodge the beast that had come behind her: you're unsure how accurate the paper's depiction of the creature is, but you can only describe it as a hedgelion, a bit larger than lions from Earth, and with various quills where its mane should be. The quills continue in a line on its back, reaching the tip of the tail.`
+			`	Having dodged the pounce, she runs past the bird, looking for cover, but the forest isn't dense enough to block the beast's advance. It attacks again, and wounds the top of one of her frontal legs. She kicks it with another leg, and readies an arrow, but it only scratches its face. The animal gets on its hind legs to swipe down at her, but is stunned, for the bird has attacked its eyes, blinding it. The huntress is swift to take aim, and an arrow pierces the creature's torso. Once it's fallen the bird flies away, and the curtains close.`
+				goto end
+			label seed
+			`	Once you have the seed in your hands, the organizers tell you to go through the path on the left, and so you do.`
+			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it, while keeping your eyes on the small stage in front of you. After the audience is all here, the curtains open, to reveal many small figures made out of what looks like paper, in a style that loosely resembles origami.`
+			`	The central figure is a young Saryd boy, who is galloping in a dense forest, as the sound effects of thunder grow increasingly frequent. Lightning strikes one of the trees on the way, and it falls in flames, blocking the boy's path, so he turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore, hurt by some of the flames and now limping through the forest.`
```

Comment:
Is this actual lightning? Or theatrics? Following on from the previous one. If these things are mostly made from some paper equivalent and its not actual Saryd in costumes, how is the effect of burning and falling trees simulated? 
Even more than the previous one, this needs the scene to be set in greater detail because lightning and trees falling in the play, when you're sitting in an actual forest clearing, makes it feel very confusing and unclear what is happening and how it is doing so.